console/module/authorization/src/main/java/org/eclipse/kapua/app/console/module/authorization/shared/model/GwtAccessRole.java

```diff
@@ -41,4 +41,11 @@ public void setRoleName(String roleName) {
         set("roleName", roleName);
     }
 
+    public String getGrantedByName() {
```

Comment:
Why not use the existing `createdBy` from `GwtUpdatableEntityModel`? There is any reason?  
The `createdBy` field in the `AccessRole` and `GwtAccessRole` represents who has created the "link" between a `User` and a `Role`, so it can be seen as "grantedBy"